buffer/src/main/java/io/netty/buffer/CompositeByteBuf.java

```diff
@@ -648,6 +660,20 @@ protected int _getUnsignedMedium(int index) {
     }
 
     @Override
+    protected int _getUnsignedMediumLE(int index) {
+        Component c = findComponent(index);
+        if (index + 3 <= c.endOffset) {
+            return c.buf.getUnsignedMediumLE(index - c.offset);
+        } else if (order() == ByteOrder.BIG_ENDIAN) {
+            return _getShortLE(index) & 0xFFFF |
```

Comment:
on the second note, I've found a bunch of places where it's mixed capitaized/lowercase. I will fix it here for now, can also submit a PR with a cleanup everyehere if you think it's a good idea..